loader/standard_loader.py

```diff
@@ -74,65 +136,77 @@ def _load_bundle(self, bundle_uuid, metadata_dict, data_objects):
         file_info_list.append(dict(uuid=metadata_file_uuid, version=metadata_file_version,
                                    name=metadata_filename, indexed=True))
 
-        # load data files by reference
-        for file_guid in data_objects:
-            file_info = data_objects[file_guid]
-            filename = file_info['name']
-            file_uuid = self._get_file_uuid(file_guid)
-            file_version = self._get_file_version(file_info)
-            cloud_urls = self._get_cloud_urls(file_info)
+        for data_file in data_files:
+            filename, file_uuid, cloud_urls, bundle_uuid, file_guid, file_version, = data_file
             logger.debug(f'Attempting to upload data file: {filename} with uuid:version {file_uuid}:{file_version}...')
-            file_uuid, file_version, filename = \
+            file_uuid, file_version, filename, already_present = \
                 self.dss_uploader.upload_cloud_file_by_reference(filename,
                                                                  file_uuid,
                                                                  cloud_urls,
                                                                  bundle_uuid,
                                                                  file_guid,
                                                                  file_version=file_version)
+            if already_present:
+                logger.info('File already present. No upload necessary.')
```

Comment:
I think this could be logged at `debug` to be consistent with the related log messages around it.
Also, I think changing to the following would be appropriate:
```
    if already_present:
             logger.debug(f'...File already present, no upload necessary:  {filename} with uuid.version {file_uuid}.{file_version}')
    else
             logger.debug(f'...Successfully uploaded data file: {filename} with uuid.version {file_uuid}.{file_version}')
```

Reply:
agreed! :+1: